samples/powerplay/src/main/cpp/PowerPlayMultiPlayer.cpp

```diff
@@ -86,18 +86,18 @@ bool PowerPlayMultiPlayer::openStream(oboe::PerformanceMode performanceMode) {
 
 
 void PowerPlayMultiPlayer::triggerUp(int32_t index) {
-    if (index >= 0 && index < mNumSampleBuffers) {
-        mSampleSources[index]->setStopMode(true);
-    }
     if (mAudioStream) {
         mAudioStream->pause();
     }
+    mSampleSources[index]->setStopMode(true);
```

Comment:
I'm a bit confused about the setStopMode/setPlayMode changes. Can you explain why each of these changes are made?

Reply:
This is actually a slight miss overall here. Ther code here is from drumthumper where multiple samples are played at once. This doesn't happen in PowerPlay so there is less need for us to go through each sample when only one is being played at a time.

Reply:
Can you test that DrumThumper still works after your change?

Reply:
This should affect drumthumper at all because I've subclassed the `iolib::SimpleMultiPlayer` which is what drumthumper uses. This class is the subclass [PowerPlayMultiPlayer.cpp](https://github.com/google/oboe/pull/2315/files/f6fc201e7d899c82454004f8fb573cf6eb9bd88b#diff-2e5d5b4fe76b7401609b97c4577e4f1dc9cc62e4907cb5113a234a357d2eea4b)